### PR TITLE
Switch bash to code component

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: docs/pages/access-controls/getting-started.mdx [hunk 1]
```diff
@@ -40,30 +29,29 @@ Members of `admin`, are full cluster administrators.
...
+```code
+$ tctl users add alice --roles=editor
 ```
 
 Once Alice signs up, she will be able to edit cluster configuration. You can list
 users and their roles using `tctl users ls`.
 
-```bash
-tctl users ls
+```code
```
Comment (r0mant) [inline docs/pages/access-controls/getting-started.mdx:39]: I'm curious, with this `code` block type, how does it know which language's syntax highlighting to apply? Is this only for shell? Or does it autodetect somehow?

**Review by r0mant — APPROVED** [review]

File: docs/pages/access-controls/getting-started.mdx (hunk 1 above)
Comment (codingllama) [reply docs/pages/access-controls/getting-started.mdx:39]: Came here to ask the same question. Do you have a reference for what the "code" block means to our particular renderers?

Adding to the above, won't the `$` interfere when copying/pasting snippets?

Comment (klizhentas) [reply docs/pages/access-controls/getting-started.mdx:39]: @codingllama @r0mant `code` block is not for general purpose programming language, it's mostly for bash code snippets, as it's very specific in it's behavior with `$` and `#`

Comment (klizhentas) [reply docs/pages/access-controls/getting-started.mdx:39]: @codingllama the plugin handles `$` and understands line continuation

**Review by russjones — APPROVED** [review]
Bot.

File: docs/pages/access-controls/getting-started.mdx (hunk 1 above)
Comment (codingllama) [reply docs/pages/access-controls/getting-started.mdx:39]: Nice, thanks.

**Pull request merged**